code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -102,7 +102,8 @@
 	for(var/reactant in required_reagents)
 		var/amt_used = required_reagents[reactant] * reaction_progress
 		var/datum/reagent/removing_reagent = holder.get_reagent(reactant)
-		total_thermal_energy += removing_reagent.get_thermal_energy() * (amt_used / removing_reagent.volume)

```

Comment:
How did this work before?

Reply:
It's the same except its just put into a var for easy debugging.

Though I'm not sure I understand the question in the first place.

Reply:
never mind, I saw it wrong